packages/Cmfcmf-React-Fiber.package/CMFHostConfigBubble.class/instance/appendChild.in..st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+appendChild: aHostNode in: aParentHostNode
+
+	(aParentHostNode children includes: aHostNode) ifTrue: [aParentHostNode children remove: aHostNode].
```

Comment:
I am fairly certain that rely on this behavior. They use [DOMNode.appendChild](https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild) which indeed reorders. It is also the only means by which the reconcilation algorithm would be able to move a keyed node to the very end of the list (except of course by moving every other item forward) since it only has insertBefore otherwise.

Hah, addendum just read about insertBefore. If passed `null` it actually appends. The react code paths are a bit challenging to exhaustively follow but at least in the place where the side effect *should* happen, it does not make use of insertBefore with a `null` reference node: https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactFiberCommitWork.new.js#L1892